spec/index.bs

```diff
@@ -1954,6 +1965,59 @@ interface GPUAdapterInfo {
     </div>
 </div>
 
+### Behaviors ### {#behaviors}
+
+A <dfn dfn>behavior</dfn> is a set of optional WebGPU functionality that may not be supported
+on all implementations, typically due user agent limitations.
+
+Functionality that is part of a behavior is available if the user agent supports it.
+In particular, there is no mechanism for an application to request that a behavior be enabled.
+
+<div algorithm>
+    A {{GPUBehaviorName}} |behavior| is <dfn dfn dfn-for=behavior noexport>enabled for</dfn>
+    a {{GPUAdapter}} |adapter| if and only if
+    |adapter|.GPUAdapter/[=behaviors=] [=list/contains=] |behavior|.
+</div>
+
+See the [[#behavior-index|Behavior Index]] for a list of the defined behaviors and their associated functionality.
+
+<h5 id=gpusupportedbehaviors data-dfn-type=interface>`GPUSupportedBehaviors`
+<span id=gpu-supportedbehaviors></span>
+</h5>
+
+{{GPUSupportedBehaviors}} is a [=setlike=] interface. Its [=set entries=] are
+the {{GPUBehaviorName}} values of the [=behaviors=] supported by an adapter.
+It must only contain strings from the {{GPUBehaviorName}} enum.
+
+<script type=idl>
+[Exposed=(Window, DedicatedWorker), SecureContext]
+interface GPUSupportedBehaviors {
+    readonly setlike<DOMString>;
```

Comment:
I'll bring this to editors to talk about whether to use a setlike or a static interface. Since we're making this a global static thing anyway, an interface probably makes sense.